### DISCUSSION TITLE
Using Calimero for a Project

### DISCUSSION THREAD

**Discussion by kevinjaypatel** [calimero-network/core#964] [Starknet Hackathon]
Hello, 

I recently participated in the Near Redacted hackathon and learned about the Calimero Network. 

I am currently working on a project where I believe Calimero can be leveraged as part of the solution. 

Is it possible to setup a call sometime to discuss more about the implementation? I would love to get your feedback. 

Best regards, 
Kevin

**Comment by chefsale** [comment]
@MatejVukosav @maksvukovic Can you guys take the call.

**Reply by maksvukovic** [reply]
Hey @kevinjaypatel - could you kindly drop your email, I'll gladly take the call!

**Reply by kevinjaypatel** [reply]
Hi @maksvukovic, thanks for your response. I just sent you a connection request on LinkedIn. Let's chat there. 

Best regards, 
Kevin
